packages/suite/src/components/wallet/WalletLayout/AccountBanners/CloseableBanner.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+import { Button, Card, H3, variables } from '@trezor/components';
+import { Translation } from 'src/components/suite';
+import { ReactNode } from 'react';
+
+interface StyledCardProps {
+    color: string;
+}
+
+const StyledCard = styled(Card)<StyledCardProps>`
+    width: 100%;
+    margin-bottom: 48px;
+    padding: 24px 28px 24px 18px;
+    border-left: 10px solid ${({ color }) => color};
```

Comment:
We already have a component `Box` (`components/src/components/Box/Box.tsx`) that does very similar think. 

Lets consolidate into using this component please.

Reply:
![image](https://github.com/trezor/trezor-suite/assets/152899911/197fc425-e77c-45e4-80af-40d30872f1cd)

Reply:
![image](https://github.com/trezor/trezor-suite/assets/152899911/e6f15a08-aef0-4a3a-bfb8-3ba7e05817c4)

Reply:
I don't know if it's worth it. Box doesnt have background property as Card, so I would need to style myself. border-left is the only property from Box which I need but I need a different color, so I would need to restyle the Box anyway.